pages/linux/hcitool.md

```diff
@@ -0,0 +1,36 @@
+# hcitool
+
+> Monitor & Configure Bluetooth connections.
+> Send special commands to Bluetooth devices.
+
+- Scan for remote devices:
```

Comment:
Just to make sure everyone knows it's about Bluetooth devices
```suggestion
- Scan for Bluetooth devices:
```

Reply:
Since this tool is only used by people who are in need
of this tool and as they definitely know that this tool is
used to work with Bluetooth, I haven't mentioned it clearly.
But definitely this is a good suggestion.

Thanks